sql/core/src/main/scala/org/apache/spark/sql/execution/arrow/ArrowConverters.scala

```diff
@@ -203,4 +205,16 @@ private[sql] object ArrowConverters {
       reader.close()
     }
   }
+
+  def toDataFrame(
```

Comment:
I had to make this public to be callable with py4j.  Alternatively, something could be added to `o.a.s.sql.api.python.PythonSQLUtils`?

Reply:
Yup, I think we should put it there.

Reply:
I left the conversion logic in `ArrowConverters` because I think there is a good chance it will change, so just added a wrapper to `PythonSQLUtils` let me know if it's ok.